README.md

```diff
@@ -1,10 +1,16 @@
-# Worldcoin trusted setup ceremony
+# Semaphore Merkle Tree Batcher (semaphore-mtb) trusted setup ceremony
 
-[World ID](https://worldcoin.org/blog/announcements/introducing-world-id-and-sdk) is a privacy-preserving proof of personhood protocol that leverages the Semaphore protocol in order to prove inclusion of members in a merkle tree where each identity commitment was generated by an orb and its corresponding private key is stored on our users' phones. In the Semaphore protocol implementation there are individual sequential insertions. Individual insertions  for big merkle trees are very expensive and therefore economically unfeasible at the World ID userbase scale ([currently nearing 2M](https://worldcoin.org/)). In order to resolve that issue we developed custom circuits written in gnark in order to do batch insertions into the Semaphore merkle tree. This gnark circuit leverages the groth16 proof system on the bn254 curve and requires a custom trusted setup ceremony to be made in order to achieve verifier soundness. 
+[World ID](https://whitepaper.worldcoin.org/proof-of-personhood) is a privacy-preserving proof of personhood protocol that leverages the [Semaphore protocol](https://semaphore.appliedzkp.org/) in order to prove inclusion of members in a merkle tree where each leaf was verified using some method (e.g. [the orb](https://whitepaper.worldcoin.org/technical-implementation)), inserted into the tree and where its corresponding private key is self-custodied by each member. In the Semaphore protocol implementation there are individual sequential insertions. Individual insertions for big merkle trees are very expensive and therefore economically unfeasible onchain operations at the World ID userbase scale (user count can be found [here](https://worldcoin.org/)). In order to scale our efforts we developed the [Semaphore Merkle Tree Batcher](https://github.com/worldcoin/semaphore-mtb) (SMTB) which is composed of custom zero-knowledge circuits written in [gnark](https://github.com/ConsenSys/gnark) in order to do batch insertions into the Semaphore merkle tree. This gnark circuit leverages the groth16 proof system on the bn254 curve and requires a custom trusted setup ceremony to be made in order to achieve verifier soundness.
```

Comment:
Still way to much emphasis on the mostly irrelevant point that it is used by WorldID. How about:

---

This repo contains the transcript of trusted setup ceremony for [Semaphore Merkle Tree Batcher](https://github.com/worldcoin/semaphore-mtb). Please see that repo for details.

To verify the transcript do

```
<commands>
```

to generate prover and verifier keys do

```
commands
```

To learn more about trusted setup ceremonies read the [BGM17] paper or the explainer by X [link]